src/Containers/PanelListWrapper/PanelListWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import { MainInterface,Main } from '@Utils/BaseStyles';
+import React from 'react';
+import styled from 'styled-components';
+import {PanelCard,IPanelCardProps} from '../PanelCard/PanelCard';
+
+export interface IPanelListWrapperProps extends MainInterface,React.HTMLAttributes<HTMLDivElement>{
+    panels?:IPanelCardProps[];
```

Comment:
Panels is required. It can be an empty array but it should never be null right? Unless you are supporting the no data available but I don't see that in your code

Reply:
well , the code handles the undefined case with a panels?.map() .... doesn't crash (Basic story)

Reply:
But it doesn't show any visual to the user. This is the same state as an empty array then...